infra/modules/domain/resources/certificates.tf

```diff
@@ -28,6 +28,8 @@ locals {
 
 # ACM certificate that will be used by the load balancer.
 resource "aws_acm_certificate" "issued" {
+  provider = aws.us-east-1
```

Comment:
CloudFront requires a cert in us-east-1, but the ALB for the service needs the cert in the same region as the ALB I believe.

So we should keep the default cert created in the default region I think. 

I don't believe the template currently makes use of CloudFront, but we could prep for that by creating another cert in us-east-1 for the domain and update the `modules/domain/data` lookup to provide the ARNs for both certs.